index.js

```diff
@@ -3,11 +3,14 @@ var EventEmitter = require('events').EventEmitter;
 var fs = require('fs');
 var sysPath = require('path');
 var asyncEach = require('async-each');
-var anymatch = require('vscode-anymatch');
+var anymatch = require('anymatch');
```

Comment:
@deepak1556 please take note that `vscode-anymatch` was added specifically to fix https://github.com/microsoft/vscode/issues/57263. We need to also bring `anymatch` to latest it seems and see if the [change](https://github.com/micromatch/anymatch/compare/master...microsoft:vscode) is still needed that I did.